kv/kvtest/kvtest.go

```diff
@@ -75,7 +74,6 @@ var testList = []struct {
 	{name: "basic", test: basic},
 	{name: "ro", test: readonly},
 	{name: "seek", test: seek},
-	{name: "increment", test: increment, txOnly: true, concurrent: true},
```

Comment:
@dennwc It wasn't passing about 25% of the time against the btree backend (in-memory). I figured that the test itself must be implemented incorrectly since surely the in-memory btree has transaction semantics. But now upon further review I see that it does not (it's marked as not safe for concurrency use)

I'll add the test back for this PR but will submit another one to fix the test. I think it might make sense to just fix btree to be concurrency safe as opposed to changing the test to not run on btree.

Reply:
Fixed it by adding the NoLocks and NoTx options for btree, missed those before!